doc/nrf/releases/release-notes-1.9.1.rst

```diff
@@ -0,0 +1,99 @@
+
+Highlights
+**********
+
+* Entry 1
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v1.9.1**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions`_.
+
+IDE support
+***********
+
+|NCS| v1.9.1 is supported by both `nRF Connect extensions for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ and :ref:`installing_ses`.
+In future releases, documentation on the setup and usage of Segger Embedded Studio Nordic Edition will no longer be available and all references to IDE support will instruct users with nRF Connect extensions for Visual Studio Code.
+In future releases, Segger Embedded Studio will no longer be tested with the build system and it is not recommended for new projects.
+
+nRF Connect extensions for Visual Studio Code will be under continued active development and supported and tested with future |NCS| releases.
+It is recommended for all new projects.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - :ref:`nrfxlib:nrf_modem_changelog`
+   * - LwM2M carrier library
+     - :ref:`liblwm2m_carrier_changelog`
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+
+.. WIP: PLACEHOLDER
+
+  See `known issues for nRF Connect SDK v1.9.0`_ for the list of issues valid for the latest release.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Trusted Firmware-M
+==================
+
+* Fixed an issue that would cause crash during boot when the RPMsg was enabled on the nRF5340 DK.
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each NCS release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``45ef0d2``, plus some |NCS| specific additions.
+This is the same commit ID as the one used for |NCS| :ref:`v1.9.0 <ncs_release_notes_190>`.
+
+For a complete list of |NCS| specific commits since v1.9.0, run the following command:
```

Comment:
Is this true?

It says since v1.9.0.

But v2.7.0, which is referenced below, was used in 1.8.0.

Reply:
This is not yet final. I wanted to double check this before the release.

Reply:
Would the correct command for this minor release be like follows, @SebastianBoe , @carlescufi ?

``git log --oneline manifest-rev ^45ef0d2``

Reply:
no, that would include commits that were in the 1.9.0 release as well.

We want

git log --oneline manifest-rev ^v2.7.99-ncs1